patientsearch/src/js/components/FilterRow.js

```diff
@@ -73,14 +108,29 @@ export default function FilterRow(props) {
     const clearDate = () => {
         setDate(null);
         setDateInput("");
-        props.onFilterChanged(dobFieldIndex, null);
+        //  props.onFilterChanged(dobFieldIndex, null, "dob");
+        props.onFiltersDidChange([
+            {
+                field: "first_name",
+                value: firstName
+            },
+            {
+                field: "last_name",
+                value: lastName
+            },
+            {
+                field: "dob",
+                value: null
+            }
+        ]);
     }
     const clearFields = () => {
         setFirstName("");
         setLastName("");
-        props.onFilterChanged(firstNameFieldIndex, "");
-        props.onFilterChanged(lastNameFieldIndex, "");
+        // props.onFilterChanged(firstNameFieldIndex, "", "first_name");
+        // props.onFilterChanged(lastNameFieldIndex, "", "last_name");
```

Comment:
Does this commented-out code still have value?
```suggestion
```

Reply:
right, no value, I did remove them.